Source/Core/Core/PowerPC/Jit64/Jit_FloatingPoint.cpp

```diff
@@ -560,6 +560,7 @@ void Jit64::fcmpx(UGeckoInstruction inst)
 {
 	INSTRUCTION_START
 	JITDISABLE(bJITFloatingPointOff);
+	FALLBACK_IF(inst.Rc);
```

Comment:
Not on Jit64.

It makes sense in the Interpreter_Tables as there is both a fcmpo and fcmpu function in Intepreter_FloatingPoint

It would seem there are no fcmpo and fcmpu functions on jit64 or jitarm64, but a fcmpx function instead.